core/scripts/local/RequestOraclePrice.js

```diff
@@ -57,8 +57,8 @@ async function run(deployedFinder, identifier, timeString) {
       await deployedFinder.getImplementationAddress(web3.utils.utf8ToHex(interfaceName.Oracle))
     );
     await deployedVoting.addSupportedIdentifier(identifierInBytes);
-    const result = await deployedVoting.requestPrice.call(identifierInBytes, timeInBN);
-    if (result == 0) {
+    const priceExists = await deployedVoting.hasPrice.call(identifierInBytes, timeInBN);
```

Comment:
I *think* you could just do deployedVoting.hasPrice() here. IIRC we need the `call` to get the return value from a non-view function, but hasPrice is a view function.

Reply:
correct